docs/DeveloperGuide.adoc

```diff
@@ -262,6 +340,26 @@ We are using `java.util.logging` package for logging. The `LogsCenter` class is
 
 Certain properties of the application can be controlled (e.g user prefs file location, logging level) through the configuration file (default: `config.json`).
 
+=== Calendar Window
+
+.Structure of the Calendar Window
+image::CalendarWindowClassDiagram.png[]
+
+The calendar window provides a visual representation of stored engagements over a monthly period.
+Users can choose to change the month being displayed and also open scrolling text windows which
+show more detailed descriptions of the stored engagements for a particular day.
+
+=== Game Window
+
+.Structure of the Game Window
+image::GameWindowClassDiagram.png[]
+
+The game window provides a simple typing game where there are moving words that users can type in order to score points.
```

Comment:
Perhaps you may consider how this feature works internally instead of what it is. This description sounds more like a User Guide. Show sequence diagrams/activity diagrams or a step by step guide on what happens internally when the game runs.